UndertaleModLib/UndertaleData.cs

```diff
@@ -509,6 +509,40 @@ public bool IsVersionAtLeast(uint major, uint minor = 0, uint release = 0, uint
             return true; // The version is exactly what supplied.
         }
 
+        /// <summary>
+        /// Sets the branch type in GeneralInfo to LTS. (true for LTS, false for non-LTS.)
+        /// </summary>
+        /// <param name="isLTS">true for LTS, false for non-LTS.</param>
+        public void SetLTS(bool isLTS)
+        {
+            if (isLTS)
```

Comment:
how is this gonna handle stuff when we have different LTS versions?

Reply:
It'll definitely still require adjustment. IsVersionAtLeast might work, or otherwise I could integrate it into an override for SetGMS2Version

Reply:
thats probably nicer yeah